test/remote/gateways/remote_safe_charge_test.rb

```diff
@@ -19,6 +19,16 @@ def setup
     @three_ds_enrolled_card = credit_card('4012 0010 3749 0014')
     @three_ds_non_enrolled_card = credit_card('5333 3062 3122 6927')
     @three_ds_invalid_pa_res_card = credit_card('4012 0010 3749 0006')
+
+    @network_token_credit_card = ActiveMerchant::Billing::NetworkTokenizationCreditCard.new({
+      brand: 'Visa',
+      payment_cryptogram: 'UnVBR0RlYm42S2UzYWJKeWJBdWQ=',
+      number: '4012001037490014',
+      source: :network_token,
+      month: '12',
+      year: 2020,
+      verification_value: '217'
```

Comment:
Why do we need to set the **month, year** and **verification_value** params? Is the network token number and cryptogram not enough? I am comparing this with [Cybersource's implementation](https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_cyber_source_test.rb#L533-L537) (which already works well) and they don't require these params. Just trying to make sure we can be consistent in supporting NT for different gateways.

Reply:
From what I remember, the request was failing if those are not provided (some sort of card validation was failing). As for the CVV, I got an explicit request from Nuvei saying: "Yes – if is provided, forward it." so I believe this answers your question.